db/re/achievement_db.conf

```diff
@@ -3121,73 +3120,73 @@ achievement_db: (
 		*1: {
 			Description: "Eliminate Morocc 50 necromancer"
 			Criteria: {
-				MobId: "Unknown_Mob - (Non-Existent)"
+				MobId: "EP14_MORS_BOSSB"
 			}
 			Goal: 50
 		}
 	}
 	Points: 50
 },*/
-/*{
+{
 	Id: 128025
 	Name: "Audience with the queen"
 	Type: "ACH_KILL_MOB_CLASS"
 	Objectives: {
 		*1: {
 			Description: "Eliminate 1 Faceworm Queen"
 			Criteria: {
-				MobId: "Unknown_Mob - (Non-Existent)"
+				MobId: "FACEWORM_QUEEN"
 			}
 			Goal: 1
 		}
 	}
 	Points: 10
-},*/
-/*{
+},
+{
 	Id: 128026
 	Name: "Warm earth"
 	Type: "ACH_KILL_MOB_CLASS"
 	Objectives: {
 		*1: {
 			Description: "Eliminate 1 Earth Faceworm Queen"
 			Criteria: {
-				MobId: "Unknown_Mob - (Non-Existent)"
+				MobId: "FACEWORM_QUEEN_G"
 			}
 			Goal: 1
 		}
 	}
 	Points: 10
-},*/
-/*{
+},
+{
 	Id: 128027
 	Name: "Water is very good exactly"
 	Type: "ACH_KILL_MOB_CLASS"
 	Objectives: {
 		*1: {
```

Comment:
Here it would be better to change how the objectives created? Instead of using `*1 `use `[1]` or use constants also example `OBJECTIVE_%d`.

example objective:

```
OBJECTIVE_1
OBJECTIVE_2
OBJECTIVE_3...
```
The maximum input value would be based on the value of `MAX_ACHIEVEMENT_OBJECTIVES`